.github/container/install_lingvo_aarch64.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -exu
+
+INSTALL_DIR="${INSTALL_DIR:-/opt}"
+LINGVO_REF="${LINGVO_REF:-HEAD}"
+LINGVO_REPO="${LINGVO_REPO:-https://github.com/tensorflow/lingvo.git}"
+
+## Install tensorflow-text
+cd ${INSTALL_DIR}
+pip install tensorflow_datasets==4.9.2 # force a recent version to have latest protobuf dep
+pip install auditwheel
+pip install tensorflow==2.13.0
+git clone http://github.com/tensorflow/text.git
+pushd text
+git checkout v2.13.0
+./oss_scripts/run_build.sh
+find * | grep '.whl$'
+pip install ./tensorflow_text-*.whl
+popd
+rm -Rf text
+
+## Install lingvo
+LINGVO_INSTALLED_DIR=${INSTALL_DIR}/lingvo
+
+[[ -d lingvo ]] || git clone ${LINGVO_REPO} ${LINGVO_INSTALLED_DIR}
+
+pushd ${LINGVO_INSTALLED_DIR}
+# Local patches, two PR waiting to be merged + one custom patch
+git fetch origin pull/326/head:pr326
+git fetch origin pull/328/head:pr328
+git config user.name "JAX Toolbox"
+git config user.email "jax@toolbox"
+git cherry-pick pr326 pr328
+cat << EOF | patch -p1
+diff --git a/lingvo/repo.bzl b/lingvo/repo.bzl
+index 55120412b..e89dabfa4 100644
+--- a/lingvo/repo.bzl
++++ b/lingvo/repo.bzl
+@@ -230,9 +230,9 @@ filegroup(
+ )
+ """,
+         urls = [
+-            "https://github.com/protocolbuffers/protobuf/releases/download/v21.9/protoc-21.9-linux-x86_64.zip",
++            "https://github.com/protocolbuffers/protobuf/releases/download/v21.9/protoc-21.9-linux-aarch_64.zip"
+         ],
+-        sha256 = "3cd951aff8ce713b94cde55e12378f505f2b89d47bf080508cf77e3934f680b6",
++        sha256 = "a584286dfa8ebb17032ece206ed74d5e9931e2edb9016e427be2a0dab3b21071"
+     )
+
+ def icu():
+EOF
+
+sed -i 's/tensorflow=/#tensorflow=/'  docker/dev.requirements.txt
+sed -i 's/tensorflow-text=/#tensorflow-text=/'  docker/dev.requirements.txt
+sed -i 's/dataclasses=/#dataclasses=/'  docker/dev.requirements.txt
+pip install -r docker/dev.requirements.txt
+pip install protobuf==3.20
+pip install patchelf
+
+# Helpful if some tests are still failing for some reason.
```

Comment:
if -> as?
Do you have some tests failing and so want to skip them for now?

Reply:
It's been a game of dependency chasing to get all the tests to pass, I got it at some point but it was still fragile, I can give it another try

Reply:
Or just update the comment to retry this when we have a more stable installation instruction?

Reply:
So test are "mostly" passing already right now, but they are flaky:
- ` //lingvo/core:saver_test` passes 100 times when run alone, but fails when it runs concurrently with other tests...
- `//lingvo/tasks/mt:model_test                                             FAILED in 100 out of 2000 in 57.6s` it's flaky with 5% of the case and the following log on failure:

```
AssertionError: 
Not equal to tolerance rtol=1e-06, atol=0.001
Mismatched value: a is different from b. 
not close where = (array([3]), array([0]))
not close lhs = [181.64984]
not close rhs = [181.65323]
not close dif = [0.00338869]
not close tol = [0.00118165]
dtype = float32, shape = (5, 2)
Mismatched elements: 1 / 10 (10%)
Max absolute difference: 0.00338869
Max relative difference: 1.8654727e-05
 x: array([[226.91527 ,  10.373269],
       [222.4018  ,   9.463906],
       [248.7229  ,   9.899419],...
 y: array([[226.91527 ,  10.373269],
       [222.4018  ,   9.463906],
       [248.72293 ,   9.89942 ],...
```

Reply:
I just excluded these two tests explicitly by patching the `bazel test` invocation, that we we'll catch regression for other tests.

Reply:
Great. They seems to be tests issue, not issue with ARM. So disabling them is good.

Reply:
I don't see the code that disable those tests. Where it is?

Reply:
I didn't amend before pushing... PTAL